src/sysrepo.c

```diff
@@ -3462,6 +3462,21 @@ sr_has_changes(sr_session_ctx_t *session)
     return 0;
 }
 
+API const struct lyd_node *
+sr_get_changes(sr_session_ctx_t *session)
+{
+    if (!session) {
```

Comment:
This is fine, I guess, I meant just adding `!session` into the `if` below.

Reply:
I thought the extra log message makes sense to differentiate the errors. Changed.